### ISSUE TITLE
can't get full properties of message

### ISSUE THREAD

**Issue by archfish** [streamnative/pulsarctl#164]
When i use pulsarctl peek message, I can not see full properties. When use pulsar-admin it work fine.

Every message was set `Content-Type` inside properties. Here is my log:

```shell
./bin/pulsarctl subscriptions peek default/public/changed LogJobConsumer
```

```log
Message ID : 17334:60:-1:-1
Properties :
{
    "publish-time": "2020-01-03T15:41:40.12+08:00"
}
Message :
00000000  7b 22 70 61 79 6c 6f 61  64 22 3a 7b 22 62 65 66  |{"payload":{"bef|
00000010  6f 72 65 22 3a 7b 7d 2c  22 61 66 74 65 72 22 3a  |ore":{},"after":|
00000020  7b 7d 2c 22 73 6f 75 72  63 65 22 3a 7b 22 6d 6f  |{},"source":{"mo|
00000030  64 65 6c 22 3a 22 4f 72  64 65 72 52 65 6c 65 74  |del":"OrderRelet|
00000040  4c 6f 67 22 2c 22 74 61  62 6c 65 22 3a 22 6f 72  |Log","table":"or|
00000050  64 65 72 5f 72 65 6c 65  74 5f 6c 6f 67 73 22 2c  |der_relet_logs",|
00000060  22 69 64 22 3a 22 35 37  66 63 61 36 34 35 2d 38  |"id":"57fca645-8|
00000070  32 66 34 2d 34 39 61 39  2d 61 37 30 38 2d 61 37  |2f4-49a9-a708-a7|
00000080  33 65 33 61 64 66 35 31  63 61 22 7d 7d 2c 22 6f  |3e3adf51ca"}},"o|
00000090  70 22 3a 22 75 22 2c 22  74 73 5f 6d 73 22 3a 31  |p":"u","ts_ms":1|
000000a0  35 37 38 30 33 37 33 30  30 31 32 30 7d           |578037300120}|
```

```shell
./pulsar-admin persistent peek-messages --subscription ReletLogJobConsumer default/public/changed 
```

```log
Message ID: 17334:44
Tenants:
{
  "Content-Type" : "application/json; charset=utf-8",
  "publish-time" : "2020-01-03T14:59:32.532+08:00"
}
         +-------------------------------------------------+
         |  0  1  2  3  4  5  6  7  8  9  a  b  c  d  e  f |
+--------+-------------------------------------------------+----------------+
|00000000| 7b 22 70 61 79 6c 6f 61 64 22 3a 7b 22 62 65 66 |{"payload":{"bef|
|00000010| 6f 72 65 22 3a 7b 7d 2c 22 61 66 74 65 72 22 3a |ore":{},"after":|
|00000020| 7b 7d 2c 22 73 6f 75 72 63 65 22 3a 7b 22 6d 6f |{},"source":{"mo|
|00000030| 64 65 6c 22 3a 22 4f 72 64 65 72 52 65 6c 65 74 |del":"OrderRelet|
|00000040| 4c 6f 67 22 2c 22 74 61 62 6c 65 22 3a 22 6f 72 |Log","table":"or|
|00000050| 64 65 72 5f 72 65 6c 65 74 5f 6c 6f 67 73 22 2c |der_relet_logs",|
|00000060| 22 69 64 22 3a 22 31 66 37 66 66 38 63 38 2d 33 |"id":"1f7ff8c8-3|
|00000070| 36 39 37 2d 34 33 37 31 2d 39 63 62 38 2d 39 39 |697-4371-9cb8-99|
|00000080| 31 33 34 30 31 36 63 31 39 65 22 7d 7d 2c 22 6f |134016c19e"}},"o|
|00000090| 70 22 3a 22 75 22 2c 22 74 73 5f 6d 73 22 3a 31 |p":"u","ts_ms":1|
|000000a0| 35 37 38 30 33 34 37 37 32 35 33 32 7d          |578034772532}   |
+--------+-------------------------------------------------+----------------+
```

**Comment by zymap** [comment]
@archfish Thank you! I will check out this issue.

**Comment by zymap** [comment]
@archfish If you need it urgently, you can fix it like how the pull request #165 fix.

**Comment by archfish** [comment]
> @archfish If you need it urgently, you can fix it like how the pull request #165 fix.

Thanks!
